backend/src/main/java/com/redhat/cloud/notifications/routers/internal/InternalResource.java

```diff
@@ -436,4 +442,30 @@ public Response unlinkDefaultBehaviorToEventType(@Context SecurityContext sec, @
             return Response.status(BAD_REQUEST).build();
         }
     }
+
+    @PUT
+    @Path("/daily-digest/time-preference/{orgId}")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    @Transactional
+    @RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_ADMIN)
+    public Response saveDailyDigestTimePreference(@Context SecurityContext sec, @NotNull LocalTime expectedTime, @PathParam("orgId") String orgId) {
```

Comment:
FYI [Quarkus offers a new set of annotations](https://quarkus.io/guides/resteasy-reactive#accessing-request-parameters) that are meant to replace the old ones from JAX-RS like `@PathParam`.

For example, you could do this:
```suggestion
    public Response saveDailyDigestTimePreference(@Context SecurityContext sec, @NotNull LocalTime expectedTime, @RestPath String orgId) {
```

Reply:
Unrelated: I don't think we need `@Context SecurityContext sec` in the method signature.

Reply:
Ok